package.json

```diff
@@ -15,29 +15,29 @@
   ],
   "private": true,
   "dependencies": {
-    "@angular/animations": "7.2.4",
-    "@angular/common": "7.2.4",
-    "@angular/compiler": "7.2.4",
-    "@angular/core": "7.2.4",
-    "@angular/forms": "7.2.4",
-    "@angular/platform-browser": "7.2.4",
-    "@angular/platform-browser-dynamic": "7.2.4",
-    "@angular/router": "7.2.4",
+    "@angular/animations": "^8.0.2",
+    "@angular/common": "^8.0.2",
+    "@angular/compiler": "^8.0.2",
+    "@angular/core": "^8.0.2",
+    "@angular/forms": "^8.0.2",
+    "@angular/platform-browser": "^8.0.2",
+    "@angular/platform-browser-dynamic": "^8.0.2",
+    "@angular/router": "^8.0.2",
     "core-js": "^2.4.1",
```

Comment:
`core-js` is no longer used dependency, could be removed from here (it will be still installed as indirect dependnecy though, but no reason to have this in `package.json` after migration to BrowserList

Reply:
```core-js```Includes polyfills for ECMAScript up to 2019 and is a must for some browsers

Reply:
I understand, but updated code no longer uses `core-js` directly. The new scaffolded apps by `angular-cli` no longer ships this by default and - as I understand - these `core-js` are added as part of new polyfilling feature we opted in here in update (once enabled via `browserlist` filter or if reverted to old behaviour):
https://github.com/angular/angular-cli/blob/4bfa4e0c06ddeee9c2c5c2be452f1f5d15796e6b/packages/angular_devkit/build_angular/src/angular-cli-files/models/es5-polyfills.js
I"m not an expert here with this change in NG 8, so up to you